Get-NetView.psm1

```diff
@@ -2306,8 +2297,8 @@ function Completion {
     which can be accessed by using "$CustomModule" as a placeholder. For example, {Copy-Item .\MyFile.txt $CustomModule}
     copies "MyFile.txt" to "CustomModule\MyFile.txt".
 
-.PARAMETER MaxThreads
-    Maximum number of simultaneous background tasks, from 1 to 16. Defaults to 5.
+.PARAMETER BackgroundThreads
```

Comment:
This would be a braking API change.  If someone was calling Get-NetView in automated form, such as SDDC* with -MaxThreads set, then said caller would break.   

Why not just leave the parameter name as MaxThreads, but use BackGround Threads as the internal variable?

Reply:
I added an alias for the old parameter name.